Source/WebCore/dom/Subscriber.cpp

```diff
@@ -111,13 +110,13 @@ void Subscriber::followSignal(AbortSignal& signal)
 
 void Subscriber::close(JSC::JSValue reason)
 {
-    auto* context = scriptExecutionContext();
+    RefPtr context = protectedScriptExecutionContext();
```

Comment:
No need for this change, we can just use `!scriptExecutionContext()` on the next line and avoid the local variable since it is only used in one place.